test/test-network/systemd-networkd-tests.py

```diff
@@ -371,7 +371,7 @@ def check_link_exists(self, link):
     def check_operstate(self, link, expected, show_status=True, setup_state='configured'):
         self.assertRegex(get_operstate(link, show_status, setup_state), expected)
 
-    def wait_online(self, links_with_operstate, timeout='20s', bool_any=False, setup_state='configured'):
+    def wait_online(self, links_with_operstate, timeout='20s', bool_any=False, setup_state='configured', setup_timeout='5'):
```

Comment:
No, please don't pass an internal parameter as a string: `setup_timeout=5`.

Reply:
i should have realized that, sorry.  changed.